feat(ShowMore): pass new state to the `onToggle` callback (#3380)
toptal/picasso: .changeset/nervous-chicken-trade.md

---
'@toptal/picasso': minor
---

---

### ShowMore

> - the `onToggle` callback has a new state as a parameter

Reviewer: ```suggestion
- add a new state as a parameter to `onToggle` callback
```
